documentation/API-Readiness-Checklist.md

```diff
@@ -2,26 +2,27 @@
 
 Checklist for api-name api-version in rx.y.
 
-| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |      | link |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      |      |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      |      |
-|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      |      |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      | Comm. release nr |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      | ICM release nr |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      | x.y.z |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |      | link |
 |  6 | User stories                                 |   O   |         O         |    O    |    M   |      | link |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |      | link |
 |  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |      | link |
 |  9 | Test result statement                        |   O   |         O         |    O    |    M   |      | link |
-| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |      |      |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |      | rx.y |
```

Comment:
Similar as previous comment.

Alternative would be to reduce line 3 to api-name.